src/index.js

```diff
@@ -28,4 +28,60 @@ function renderTasks() {
     taskList.appendChild(listItem);
   });
 }
+
+function addTask(description) {
+  const newTask = {
+    description,
+    completed: false,
+    index: tasks.length,
+  };
+  tasks.push(newTask);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  renderTasks();
+}
+
+function deleteTask(index) {
+  tasks.splice(index, 1);
+  // eslint-disable-next-line no-return-assign
```

Comment:
- [ ] Hey, great job so far! Please, do remember that the linters check helps us improve our development skill by writing codes that conform to the linter. please, kindly remove all the eslint disable from your codebase and try writing your codes to conform to the linter.